pages/en/lb4/Testing-your-application.md

```diff
@@ -270,12 +274,12 @@ describe('ProductController (unit)', () => {
     it('retrieves details of a product', async () => {
       const controller = new ProductController(repository);
       const findStub = repository.find as sinon.SinonStub;
-      findStub.resolves([{id: 1, name: 'Pen'}]);
+      findStub.resolves([{name: 'Pen', slug: 'pen'}]);
 
-      const details = await controller.getDetails(1);
+      const details = await controller.getDetails('pen');
```

Comment:
@shimks A general question: do we have an example package contains all these code? 
I came across the thought when searched for function `getDetails`, it should be defined in some folded snippet. Just feel if we have an example testing package might be more straight-forward.(out of the scope of your PR definitely)

Reply:
I've actually went ahead and created an app to make sure these codes work. These code snippets are literally code snippets form the project that I've made :)